packages/apps/docs/src/components/Markdown/MDNotification/style.css.ts

```diff
@@ -0,0 +1,12 @@
+// TODO: use common wrapper once https://github.com/kadena-community/kadena.js/pull/735/
+//       gets merged
+import { sprinkles } from '@kadena/react-ui/theme';
+
+import { style } from '@vanilla-extract/css';
+
+export const wrapperClass = style([
+  sprinkles({
+    marginY: '$5',
+    marginX: 0,
+  }),
+]);
```

Comment:
Can you apply these styles in the same way that you did in the other PR? Then I think they will get resolved in the merge by default (I would expect)